source/shared/net/desktop-wallet/change-restake-settings.rst

```diff
@@ -16,7 +16,7 @@ By default, your baker rewards are restaked when you create the baker account. H
 
 #. Go to **Multi Signature Transactions**, select **Make new proposal**, and then select **Update baker restake earnings**.
 
-#. Select the **Identity** that's associated with the baker account whose keys you want to update, and then select **Continue**.
+.. #. Select the **Identity** that's associated with the baker account whose keys you want to update, and then select **Continue**.
```

Comment:
Same thing again. The list numbering is off here as well.

![image](https://user-images.githubusercontent.com/76045015/123597426-9e6e8500-d7f3-11eb-9c76-044127663006.png)